rosdep/python.yaml

```diff
@@ -5736,10 +5736,16 @@ python3-docstring-parser:
     pip:
       packages: [docstring-parser]
   debian:
+    '*':
+      pip:
+        packages: [docstring-parser]
```

Comment:
I believe in this case, that you would want to create a `python3-docstring-parser-pip` package and depend on that.

We keep the dependencies explicit as you can't build a binary debian against pip dependencies.

Reply:
Actually, this is the correct way to do this.